packages/cli/src/config/config.ts

```diff
@@ -48,7 +48,7 @@ interface CliArgs {
   telemetryTarget: string | undefined;
   telemetryOtlpEndpoint: string | undefined;
   telemetryLogPrompts: boolean | undefined;
-  allowed_mcp_server_names: string | undefined;
+  'allowed-mcp-server-names': string | undefined;
```

Comment:
Should these map 1:1 to the actual user typed name or should they be TypeScripty? I.e. should this be `allowedMCPServerNames`?

Reply:
they should probably be TypeScripty, good catch @NTaylorMullen 👍 

Is this something the lint job should catch?